### ISSUE TITLE
ios7 and ios8 warnings

### ISSUE THREAD

**Issue by mousebird** [mousebird/WhirlyGlobe#475]
Bump the test project up to ios8 and fix the various warnings.
Lots of sizeWithFont: and NSGregorianCalendar.

**Comment by mousebird** [comment]
I think there might be some new warnings from Xcode too.  See if you can test against the latest.

**Comment by rghosh0** [comment]
For this one, I believe that you've fixed most of those warnings, Steve?  I saw reference to some changes targeting warnings in the git log. I removed the few that were left, except that one is still outstanding: I still have to create a launch or storyboard xib for the test app.

**Comment by mousebird** [comment]
Try increasing the target iOS version.  I think a lot of them only appear when you target, say, iOS8.

**Comment by rghosh0** [comment]
Rats, I'm still not seeing them. I changed all of the Deployment Targets to 8.0 (for Project and Target, for WhirlyGlobeComponentTester, WhirlyGlobe-MaplyComponent, and WhirlyGlobeLib).

I see in project settings that there are various options for what warnings to report, but I tried changing them and they change the project files, which would presumably propagate to the github repo, so I would think I'd have the same settings as you.
